cmd/internal/db/locode.go

```diff
@@ -0,0 +1,31 @@
+package locodedb
+
+import (
+	"fmt"
+	"os"
+)
+
+func panicOnPrmValue(n string, v any) {
+	panic(fmt.Sprintf("invalid parameter %s (%T):%v", n, v, v))
+}
+
+// CsvDB is a resulting database in CSV format. Path should be a valid path to the directory.
+type CsvDB struct {
+	Path string
+}
+
+func New(path string) CsvDB {
+	switch {
+	case path == "":
+		panicOnPrmValue("Path", path)
+	}
+	_, err := os.Stat(path)
+
+	if os.IsNotExist(err) {
+		panicOnPrmValue("Output directory does not exist", path)
```

Comment:
are you sure u want `"invalid parameter Output directory does not exist (string):some_path"` message here?